config/locales/de.yml

```diff
@@ -517,6 +517,11 @@ de:
       spree/zone:
         one: Gebiet
         other: Gebiete
+  date:
```

Comment:
All these date localizations are provided by the `rails-i18n` gem. We don't want to maintain them in the Solidus translation file. They effect more than Solidus. 

We even have a note about that in the readme.

Reply:
ahh sorry, i just forget to add rails i18n. this makes total sense. i chengd the commit accordingly